build.gradle

```diff
@@ -72,42 +73,67 @@ dependencies {
 	testCompile 'org.mockito:mockito-all:2.0.2-beta'
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: ':shadowJar') {
+def assetsDirectory = file("${buildDir}/assets")
+git {
+	assets {
+		url 'https://github.com/triplea-game/assets.git'
+		directory assetsDirectory
+		branch 'master'
+		singleBranch true
+	}
+}
+import com.install4j.gradle.Install4jTask
+
+task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
+}
+
+
+task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
+	ext.output = file("$project.buildDir/libs/all/triplea.jar")
+	from shadowJar.archivePath
+	into ext.output.parent
+	rename shadowJar.archivePath.name, output.name
+}
+
+task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
 	classifier 'all_platforms'
-	['assets', 'dice_servers', 'doc', 'license', 'maps', 'old'].each { folder ->
+	['assets', 'dice_servers', 'license', 'maps', 'old'].each { folder ->
```

Comment:
dice servers is probably good here, since the allPlatform is deployed to the dice servers/lobby, etc..